overrides/macos-override.json

```diff
@@ -31,7 +31,13 @@
             "state": "enabled"
         },
         "autofill": {
-            "state": "enabled"
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "containerstore.com",
+                    "reason": "Poor UX: https://app.asana.com/0/1200930669568058/1202105131841960/f"
```

Comment:
Let's keep Asana links out of the config. There is potential one day in the future these reasons could be user facing.

Reply:
Done!